src/TableV8/styled.tsx

```diff
@@ -21,93 +21,130 @@ import clsx from 'clsx';
 import isPropValid from '@emotion/is-prop-valid';
 import { styled } from '../ThemeProvider';
 import colors from '../theme/defaultTheme/colors';
+import { DnaLoader } from '../DnaLoader';
 
 export const TABLE_CLASSES = {
+  LOADING_COMPONENT: 'rt-loading',
+  RESIZABLE_HEADER_CONTENT: 'rt-resizable-header-content',
+  RESIZER: 'rt-resizer',
+  SORT_BUTTON: 'rt-sort-button',
   TABLE_CONTAINER: 'rt-table-container',
   TABLE: 'rt-table',
   TBODY: 'rt-tbody',
+  TD_WRAPPER: 'rt-td-wrapper',
   TD: 'rt-td',
+  TH_WRAPPER: 'rt-th-wrapper',
   TH: 'rt-th',
   THEAD: 'rt-thead -header',
   TR_GROUP: 'rt-tr-group',
   TR: 'rt-tr',
-  RESIZABLE_HEADER_CONTENT: 'rt-resizable-header-content',
 };
 
 const COLORS = {
-  BACKGROUND: colors.grey_4,
+  BACKGROUND_ALT: colors.grey_4,
```

Comment:
Small thing... other components/styling so far uses "secondary" instead of ALT if i understand this context right